spyderlib/widgets/sourcecode/base.py

```diff
@@ -61,15 +63,28 @@ def setup_appearance(self, size, font):
         self.setFont(font)
         
     def show_list(self, completion_list, automatic=True):
+        types = [c[1] for c in completion_list]
+        completion_list = [c[0] for c in completion_list]
         if len(completion_list) == 1 and not automatic:
             self.textedit.insert_completion(completion_list[0])
             return
-        
+
         self.completion_list = completion_list
         self.clear()
-        self.addItems(completion_list)
+
+        icon_lut = {'instance': 'attribute.png',

```

Comment:
What does `lut` mean? I don't get it :-)

What about renaming this variable for `icons_map`?